todo_app/data/trello_items.py

```diff
@@ -0,0 +1,33 @@
+import os
+import requests
+from todo_app.item import Item
+
+board_url = "https://api.trello.com/1/boards/6212689e46f57218f07af552/lists/open"
+items_url = "https://api.trello.com/1/cards"
+
+def get_items(key, token):
+    items = []
+    querystring = { "key": key,
+                    "token":token,
+                    "cards":"open"}
+    response = requests.request("GET", board_url, params=querystring)
+    response_json = response.json()     
+    for trello_list in response_json:
+        for card in trello_list['cards']:
+            items.append (Item.from_trello_card(card, trello_list))
+    return items
+
+def add_item(key, token, list_id, title):
+    querystring = { "key": key,
+                    "token":token,
+                    "idList":list_id,
+                    "name":title }
+    response = requests.request("POST", items_url, params=querystring)
+    return None
+
+def complete_item(key, token, list_id, item_id):
+    querystring = { "key": key,
+                    "token":token,
+                    "idList":list_id}
+    response = requests.request("PUT", items_url + '/' + item_id, params=querystring)
+    return None    
```

Comment:
No harm in this line of code, but it's identical to having no return statement at all